surface/include/pcl/surface/mls.h

```diff
@@ -52,6 +52,28 @@
 
 namespace pcl
 {
+  /** \brief Data structure used to store the results of the MLS fitting
+    * \note Used only in the case of VOXEL_GRID_DILATION or DISTINCT_CLOUD upsampling
```

Comment:
I don't think this comment is true. VOXEL_GRID_DILATION or DISTINCT_CLOUD set the the caching procedure to true. But the other methods, also use caching as long as the user manually sets it.